commit/merkleroot/outcome_test.go

```diff
@@ -36,78 +34,3 @@ func Test_buildReport(t *testing.T) {
 		}
 	})
 }
-
-func Test_fChainConsensus(t *testing.T) {
```

Comment:
Maybe we can copy/paste this test to the shared folder?

Reply:
done.